test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,41 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+
+    @Test
+    public void isAnyNull() {
+        // no null
+        assertNoNull("123", 12);
+        assertNoNull("");
+
+        // only one null
+        assertHasNull("1232", null, 1009);
+        assertHasNull(10, null);
+        assertHasNull(null, 1223);
+
+        // multiple nulls
+        assertHasNull(20.0, null, null);
+        assertHasNull("Joks", null, null, null, null);
+        assertHasNull(null, null, null, "pop");
+
+
+    }
```

Comment:
Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.